neoforge/src/main/java/dev/nolij/zume/neoforge/NeoZume.java

```diff
@@ -114,9 +112,9 @@ public NeoZume(IEventBus modEventBus, ModContainer modContainer) {
 			//noinspection unchecked
 			NeoForge.EVENT_BUS.addListener((Class<? extends Event>) RENDER_FRAME_EVENT, this::render);
 		} else if (
-			RENDER_TICK_EVENT != null && 
-			RENDER_TICK_EVENT_PHASE_GETTER != null && 
-			TICK_EVENT_PHASE_START != null) {
+			RENDER_TICK_EVENT != null &&
+				RENDER_TICK_EVENT_PHASE_GETTER != null &&
```

Comment:
Remove the extra indentation, but keep the whitespace stripping at the end of the line.